Sources/SwiftDocC/Infrastructure/DocumentationContext.swift

```diff
@@ -621,9 +624,19 @@ public class DocumentationContext: DocumentationContextDataProviderDelegate {
                 // Add to document map
                 documentLocationMap[url] = technologyResult.topicGraphNode.reference
                 
-                let sourceLanguage = SourceLanguage.swift
-                let node = DocumentationNode(reference: technologyResult.topicGraphNode.reference, kind: .technology, sourceLanguage: sourceLanguage, name: .conceptual(title: technology.intro.title), markup: technology.originalMarkup, semantic: technology)
-                documentationCache[technologyResult.topicGraphNode.reference] = node
+                let reference = technologyResult.topicGraphNode.reference.withSourceLanguages(sourceLanguages)
+                
+                let node = DocumentationNode(
+                    reference: reference,
+                    kind: .technology,
+                    sourceLanguage: Self.defaultLanguage(in: sourceLanguages),
```

Comment:
Nit since I don't think this is directly related to this PR. But this should this be `defaultSourceLanguage`?

```suggestion
                    defaultSourceLanguage: Self.defaultLanguage(in: sourceLanguages),
```

Reply:
Hmm, possibly. Out of scope of this PR though, especially as `sourceLanguage` is public API. Thanks for bringing it up!